ext/crypto/00_crypto.js

```diff
@@ -1299,12 +1299,10 @@
           }
 
           const hashAlgorithm = key[_algorithm].hash.name;
-          const saltLength = normalizedAlgorithm.saltLength;
           return await core.opAsync("op_crypto_verify_key", {
             key: keyData,
             algorithm: "RSA-PSS",
             hash: hashAlgorithm,
-            saltLength,
             signature,
           }, data);
```

Comment:
This doesn't seem right.

Reply:
This change will fly under the WPT radar because it only performs a round-trip test with the same parameters.

It is however expected that when signed with saltLength of a given value, only verify with a the same value will succeed. With the rsa update in place that's not the case.

Reply:
Please add the following test to `cli/tests/unit/webcrypto_test.ts`

```ts
// https://github.com/denoland/deno/pull/16327#pullrequestreview-1144702051
Deno.test(async function pssSaltLengthRoundTrip() {
  const keyPair = await crypto.subtle.generateKey(
    {
      name: "RSA-PSS",
      modulusLength: 2048,
      publicExponent: new Uint8Array([1, 0, 1]),
      hash: "SHA-256",
    },
    true,
    ["sign", "verify"],
  );

  const data = crypto.getRandomValues(new Uint8Array(16));
  const algorithm = { name: "RSA-PSS", saltLength: 20 };
  const sig = await crypto.subtle.sign(algorithm, keyPair.privateKey, data);

  assert(await crypto.subtle.verify(algorithm, keyPair.publicKey, sig, data));
  assertEquals(
    await crypto.subtle.verify(
      { ...algorithm, saltLength: 32 },
      keyPair.publicKey,
      sig,
      data,
    ),
    false,
  );
});
```

Reply:
That may be a valid test. But it fails on `main` and this patch has no affect on it.

Reply:
You're correct that this indeed also happens on `main` too. That does not however make this an acceptable state. I've opened an [issue](https://github.com/denoland/deno/issues/16362) to track this.